ledger/participant-state/kvutils/src/test/lib/scala/com/daml/ledger/participant/state/kvutils/KVTest.scala

```diff
@@ -430,13 +431,15 @@ object KVTest {
       submitter: Ref.Party,
       commandId: Ref.CommandId,
       deduplicationTime: Duration,
-      recordTime: Timestamp,
+      submissionId: Ref.SubmissionId,
   ): SubmitterInfo =
     SubmitterInfo(
       actAs = List(submitter),
       applicationId = Ref.LedgerString.assertFromString("test"),
       commandId = commandId,
-      deduplicateUntil = recordTime.addMicros(deduplicationTime.toNanos / 1000).toInstant,
+      deduplicationPeriod = DeduplicationPeriod.DeduplicationDuration(deduplicationTime),
+      submissionId = submissionId,
+      ledgerConfiguration = null,
```

Comment:
Elsewere we just don't pass unset fields to scalapg's constructors; is there a reason why it's not done here? If not, can you remove it here and elsewhere?

Reply:
Set it to null because this is not scalapb code, but our own model

Reply:
Why are we setting nulls in our own models?

Reply:
Update to use some defaults